front-end/LoanCard/LoanCardStyle.js

```diff
@@ -0,0 +1,70 @@
+import { StyleSheet } from 'react-native'
+const style = StyleSheet.create({
```

Comment:
@JialinWang1 let's simplify the styles we need to create with using the card from here:
https://docs.nativebase.io/Components.html#card-showcase-headref